engine/conditions.go

```diff
@@ -0,0 +1,66 @@
+package engine
+
+import "fmt"
+
+type ConndtionResponse struct {
```

Comment:
hmm looks like the commit that fixed these was overwritten,  `ConndtionResponse` => `ConditionResponse`